app/controllers/posts_controller.rb

```diff
@@ -3,7 +3,14 @@ class PostsController < ApplicationController
   before_action :set_post, only: %i[show destroy]
 
   def index
-    @posts = Post.includes(:comments).paginate(page: params[:page], per_page: 10)
+    if params[:user_id]
+      @user = User.find(params[:user_id])
+      @posts = @user.posts.includes(:comments).paginate(page: params[:page], per_page: 10)
+    else
+      @posts = Post.includes(:comments).paginate(page: params[:page], per_page: 10)
+    end
+
+    render json: @posts
```

Comment:
- [ ] Great work so far but I guess there is a misunderstanding :fearful: because your API endpoint for getting all the posts is working but the rest of your app should work the same as it was before because right now I am not able to get Post Index page :fearful: Kindly fix this issue to make your app work as it was working before in the previous milestone :+1:

###  :green_circle: IMPORTANT :memo: -
Kindly follow the project requirements because your app should work the same means all the views should be accessible as they were before :+1: and the API endpoints controller/methods should be separate from these views controllers :thinking: so kindly fix the issue to make your app follow the project requirements :+1: .